doc/importexport.rst

```diff
@@ -76,8 +76,114 @@ There are no parameters for this method other than the file path to import.
 
     result = cq.importers.importStep('/path/to/step/block.stp')
 
+Exporting STEP
+###############
+
+This section covers exporting CadQuery Workplane objects to STEP. For exporting assemblies to STEP, see the next section.
+
+Default
+--------
+
+The exporters module handles exporting Workplane objects to STEP. It is not necessary to set the export type explicitly
+since it will be determined from the file extension. Below is an example.
+
+.. code-block:: python
+    # Create a simple object
+    box = cq.Workplane().box(10, 10, 10)
+
+    # Export the box
+    cq.exporters.export(box, "/path/to/step/box.step")
+
+Non-Default File Extensions
+----------------------------
+
+If there is a requirement to export the STEP file using an "stp" extension, CadQuery will throw an error saying that it does
+not recognize the file extension. In that case the export type has to be specified.
+
+.. code-block:: python
+    # Create a simple object
+    box = cq.Workplane().box(10, 10, 10)
+
+    # Export the box
+    cq.exporters.export(box, "/path/to/step/box.stp", cq.exporters.ExportTypes.STEP)
+
+Setting Extra Options
+----------------------
+
+There are multiple options that can be set when exporting an object to a STEP file, but those can only be accessed by calling
+the :py:meth:`Shape.exportSTEP`` method on the lower level CadQuery object. For an explanation of the options available, see
```

Comment:
```suggestion
the :py:meth:`Shape.exportSTEP`` method on the lower level CadQuery object or the :py:meth:`Assembly.exportAssembly`` method. For an explanation of the options available, see
```

IS this what you mant @lorenzncode ?

Reply:
@adam-urbanczyk Yes, partly.  I would suggest also to remove this statement: 
    "but those can only be accessed by calling the :py:meth:`Shape.exportSTEP`` method on the lower level CadQuery object."

It is incorrect, because you can do e.g.:

`assy.save('out.step', write_pcurves=False)`

Reply:
OK, could you update yourself then?

Reply:
I'll add a few changes based on your latest commit.